recipes/cryptopp/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "cryptopp" %}
+{% set version = "5.6.5" %}
+{% set sha256 = "79fd5514b3b191a1c6d934cd989d5e058f4726a72a3dad2444bd1274a6aae686" %}
+
+package:
+  name: {{ name| lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name | upper }}-{{ version | replace('.', '_') }}.tar.gz
```

Comment:
FYI this doesn't need to match the name of the file online. This is just what the file is saved as. So it could just as well be `{{ name }}-{{ version }}.tar.gz` FWIW. Not that it needs to be. Just noting this.